src/luascript.cpp

```diff
@@ -17113,11 +17139,7 @@ int LuaScriptInterface::luaCreatureEventOnCallback(lua_State* L)
 	// creatureevent:onLogin / logout / etc. (callback)
 	CreatureEvent* creature = getUserdata<CreatureEvent>(L, 1);
 	if (creature) {
-		if (!creature->loadCallback()) {
-			pushBoolean(L, false);
-			return 1;
-		}
-		pushBoolean(L, true);
+		pushBoolean(L, creature->loadCallback());
```

Comment:
It's not bad, but why do you have to modify things that have nothing to do with the changes you mention?